tests/test_api.py

```diff
@@ -338,6 +338,37 @@ def test_properties(self):
         self.assertEqual(self.dbus_props.Get('org.freedesktop.Test.Other', 'color'),
                          'yellow')
 
+        changed_props = []
```

Comment:
Thanks! Please squash this commit into the previous one that adds the API.

Reply:
Squashed.